helper/testutil/k8s.go

```diff
@@ -0,0 +1,30 @@
+package testutil
+
+import (
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// LBService returns a Kubernetes service of type LoadBalancer.
+func LBService(name, lbIP string) *apiv1.Service {
```

Comment:
I'm not a big fan of having this in a shared package. I'd prefer to see it duplicated in each test package because it's easier to modify if a specific package needs to change its LB and it's not a lot of code.

Reply:
Ok, yeah I think that seems to be the coding preference at HashiCorp. I've removed this helper and made it more "WET" 😁